docs/src/examples/langchain.md

```diff
@@ -0,0 +1,138 @@
+# Code Documentation Q&A Bot 2.0
```

Comment:
this all references lance format

Reply:
I'll remove this example and fix in another PR